src/ProjectTemplates/BlazorTemplates.Tests/BlazorWasmTemplateTest.cs

```diff
@@ -131,6 +125,8 @@ public async Task BlazorWasmHostedTemplate_Works(BrowserKind browserKind)
             {
                 EnsureBrowserAvailable(browserKind);
             }
+            
+            Assert.True(1 == 0);
```

Comment:
??

Reply:
Its really hard to see the logs without a failing test in the run UI, so I always break a test, the last thing I'm trying to chase down is where the webapp output went, I got the playwright logging to show up in the log file, but the debug output from the server is gone now